webots_ros2_tiago/launch/tiago.launch.py

```diff
@@ -34,19 +34,33 @@ def generate_launch_description():
     webots = launch_ros.actions.Node(package='webots_ros2_core', node_executable='webots_launcher',
                                      arguments=arguments, output='screen')
     # Controller node
-    synchronization = launch.substitutions.LaunchConfiguration('synchronization', default=False)
+    synchronization = launch.substitutions.LaunchConfiguration(
+        'synchronization', default=False)
```

Comment:
We should probably update all the `test_flake8.py` test files to have a line length of 128 as you did here:
https://github.com/cyberbotics/epuck_ros2/blob/1d51f52f575a9cf57d2871aa7de5cf0012edc57f/epuck_ros2/test/test_flake8.py#L23

Reply:
That was my editor :/ Not sure why CI didn't raise an error, but I will increase